### DISCUSSION TITLE
How to access a CLI utility installed via pip, through tea?

### DISCUSSION THREAD

**Discussion by urcades** [pkgxdev/pkgx#466] [Help]
I've recently installed [instascrape](https://github.com/tnychn/instascrape) via pip, through tea, and I'm not sure how to access the resulting cli app. 

When I attempt to use `instascrape` as intended, i.e. via the repo's example —

```
$ instascrape down @emmawatson -timeline -PF "likes_count >= 1000000" -l 100 --preload -d ~/Pictures/Instagram
```

— I receive an error `tea: command not found: instascrape`.

If anyone could help me figure out how to access the instascrape cli with tea installed, I'd appreciate it.

**Comment by getneil** [comment]
@urcades hi have you tried executing `instscraper` not `instascrape`?
<img width="932" alt="Screenshot 2023-03-23 at 8 04 25 PM" src="https://user-images.githubusercontent.com/7913978/227198523-b07bf43e-88bf-4715-9a6c-7710207d0bcb.png">

**Comment by urcades** [comment]
@getneil I've tried both names — this project is confusing in that its pip package is named `instascraper` while the invocation uses `instascrape` — regardless, neither command yields any result

**Comment by mxcl** [comment]
As long as your pip was installed within the last week the tools go to `~/.local/bin` which tea’s magic will add to your PATH.

Otherwise they will be buried in the tea prefix (`~/.tea/python.org/v3.11.2/bin` probs)

I would delete `~/.tea/pip.pypa.io` and reinstall.

Also I imagine there is a way to get pip to be verbose in its output which would tell you where it puts things though I don't know what the trick is there.
